app/controllers/users_controller.rb

```diff
@@ -1,21 +1,25 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
+  before_action :set_user
+
   def index
     @users = User.with_attached_avatar.order(:id).page(params[:page])
   end
 
-  def show
-    @user = User.target_user(params)
-  end
+  def show; end
 
   def followees
-    @user = User.target_user(params)
     @followees = @user.followees
   end
 
   def followers
-    @user = User.target_user(params)
     @followers = @user.followers
   end
+
+  private
+
+  def set_user
+    @user = User.find_by(id: params[:id])
```

Comment:
```suggestion
    @user = User.find(params[:id])
```

にしないと、`@user`がnilになるケースの考慮が必要になります